src/main/scala/beam/utils/scenario/BeamScenarioLoader.scala

```diff
@@ -205,6 +213,16 @@ class BeamScenarioLoader(
     population
   }
 
+  private def selectedPlanFromPersonOrBuildNew(person: Person): Plan = {
+    var result = person.getSelectedPlan
```

Comment:
Why using `var`?  You could use `Option(person.getSelectedPlan).getOrElse{}`

Reply:
replaced